composer.json

```diff
@@ -27,6 +27,9 @@
         "sort-packages": true
     },
     "extra": {
+        "branch-alias": {
+            "dev-master": "2.1.x-dev"
+        },
```

Comment:
Could we remove that again please? It is prone to be forgotten because it is not part of our release process. If somebody needs a specific unreleased fix `composer req typo3-contentblocks/composer-plugin:dev-master#0f5d83a0cb7ed7b832eced82541d57e2577de2be` works just as well IMHO.

After that I am fine with the change. Thank you very much!

Reply:
Sure.
Sorry for the quick-fixes. I needed to install my contentblocks in a project.